code/modules/clothing/under/miscellaneous.dm

```diff
@@ -162,8 +162,8 @@
 	item_color = "officer"
 	displays_id = 0
 
-/obj/item/clothing/under/rank/centcom/magistrate/New()
-	..()
+/obj/item/clothing/under/rank/centcom/magistrate/Initialize(mapload)
+	. = ..()
 	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Magistrate\" and bears [station_name()] on the left shoulder."
```

Comment:
dont need to override the desc in init, just set the desc in the type definition. and remove the override.

Reply:
The value of `station_name()` is not available at compile-time, so has to be deferred to until after the map is known.